woocommerce/payment-gateway/admin/class-sv-wc-payment-gateway-admin-user-edit-handler.php

```diff
@@ -56,6 +56,8 @@ class SV_WC_Payment_Gateway_Admin_User_Edit_Handler {
 	/**
 	 * Initialize and setup the Admin User Edit Handler
 	 *
+	 * TODO: $text_domain is deprecated, remove argument on next breaking release
+	 *
 	 * @since 3.0.0
 	 */
 	public function __construct( $plugin, $text_domain ) {
```

Comment:
Right, that's what I did originally in 2b99b7d6c0cb962f4c9abbe76bf3185986e6f1ac, but reverted it in ae1b0412fb4260ae2cbc7dd2cc9b2f8c22d51d50 so that this would not be a breaking change. We discussed that in Slack: https://skyverge.slack.com/archives/woocommerce/p1437688625001115